src/app/bundles/bundle-page/bundle-page.actions.ts

```diff
@@ -0,0 +1,43 @@
+import { Action } from '@ngrx/store';
+import { BundleDocument } from '../shared/bundle.interfaces';
+
+export const LOAD_BUNDLE_DOCUMENTS_SUCCESS = '[c2i] load bundle page actions';
+
+export class LoadBundleDocumentsSuccess implements Action {
```

Comment:
The classes don't do anything, personally I would just use an interface and implement them with plain JS objects:

```
export interface LoadBundleDocumentsSuccess extends Action {
  type: LOAD_BUNDLE_DOCUMENTS_SUCCESS,
  payload: BundleDocument[]
}
```

Probably just my personal preference, because it doesn't really matter either way.

Reply:
not sure whether we can instantiate interfaces with new

Reply:
you can't you would just do ```
const action = {
  type: LOAD_BUNDLE_DOCUMENTS_SUCCESS,
  payload: ...
}
```